### PR TITLE
Add kafka config for how long does kafka query server topic will trigger timeout

### PR DESCRIPTION
**Pull request by zifeihan**
…ger timeout.

Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
Add kafka config for how long does kafka query server topic will trigger timeout

### CODE REVIEW

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaProducerManager.java [hunk 1]
```diff
@@ -42,6 +46,8 @@
 
     private KafkaProducer<String, Bytes> producer;
 
+    private static final ILog LOGGER = LogManager.getLogger(KafkaProducerManager.class);
```
Comment (wu-sheng) [inline apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaProducerManager.java:49]: The static field should be on the top.

**Review by wu-sheng — APPROVED** [review]
LGTM. @dmsolr Please recheck.

**Review by dmsolr — APPROVED** [review]

**Comment by wu-sheng** [comment]
It seems like, we could set the timeout too quickly.

**Review by wu-sheng — CHANGES_REQUESTED** [review]
Kafka e2e timeout, waiting for adjusting the configuration in the test.

**Review by dmsolr — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**